src/asset-handler.js

```diff
@@ -974,13 +975,15 @@ var assetHandler = (function() {
                                     return callback([]);
                             }
                         }
+                    // eslint-disable-next-line no-unused-vars
                     } catch (e) {/*parse error*/}
```

Comment:
We can resolve this without an eslint ignore - see: https://github.com/ccnmtl/mediathread-chrome/pull/435#discussion_r1718524551